actor/image_upload/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "image_upload"
+version = "0.1.0"
+authors = [ "wasmCloud Team" ]
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+name = "image_upload"
+
+[dependencies]
+async-trait = "0.1"
+form_urlencoded = "1.0"
+futures = "0.3"
+image = { version = "0.24.1", default_features = false, features = [ "gif", "jpeg", "ico", "png", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "dds", "farbfeld", "jpeg_rayon" ] }
```

Comment:
does this no longer require bindgen?

Reply:
yes no bindgen - lost one of the file type detectors, but it still has the common types.